libs/order/README.md

```diff
@@ -0,0 +1,7 @@
+# @daffodil/order
```

Comment:
This PR is pretty big. Can you truncate the changes to checkout/demo and do those in a separate PR?

I'd like this PR to only reflect the addition of `@daffodil/order` and some models/actions. The other changes can happen atomically in smaller commits & PRs.

Reply:
I have removed demo/checkout changes from this PR. Note that only the actions and driver interface actually contain functional changes from the old order module in the checkout package.